.github/workflows/ci.yml

```
@@ -77,6 +77,83 @@ jobs:
           # check includes jacocoTestCoverageVerification (95% thresholds)
           gradle --no-daemon -q check
 
+      - id: coverage-summary
+        name: Coverage Summary (Job Summary)
+        if: always()
+        shell: bash
+        run: |
+          python3 - <<'PY'
+import os
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 86: could not find expected ':'

(syntax-check)